microprofile/tests/tck/tck-health/pom.xml

```diff
@@ -31,6 +31,16 @@
     <name>Helidon Microprofile Tests TCK Health</name>
 
     <dependencies>
+        <dependency>
+            <groupId>io.helidon.microprofile.health</groupId>
+            <artifactId>helidon-microprofile-health</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
```

Comment:
Metrics seem to be added to all poms, including the arqulilian. Please explain

Reply:
See the first comments. It is required if you use Helidon Arquillian.